optashift-employee-rostering-gwtui/pom.xml

```diff
@@ -119,6 +123,63 @@
           <moduleName>org.optaplanner.openshift.employeerostering.gwtui.WorkerRosteringWebapp</moduleName>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
I wonder if there's a plugin that just does all this magic with a much less verbose configuration.
I looked for a patternfly plugin, but that doesn't exist (and it doesn't need to because patternfly is just a webjar).
So I looked for a webjars plugin, for which at least one plugin exists, but it's unclear if it's stable enough to use.

So, let's merge this PR and let's open a separate issue to research/investigate:
- confirm that the webjars is a consistently used standard. So that Central Maven has much many of such webjars.
- Try the use of a webjars maven plugin. Which one?

Reply:
Yeah, it's too much configuration for just a dependency.. Good idea, let's make it a separate issue.